### PR TITLE
Fix after restore failed

### PR DESCRIPTION
**Pull request by pengweisong**
Before restore, we will backup nebula data path in local, then when restore failed, we should mv them back and restart all services.
Add fix process when restore failed:
  - check if all alive
  - stop all services
  - move the data folder back if have
  - start all service

### CODE REVIEW

**Review by critical27 — DISMISSED** [review]

File: cmd/restore.go [hunk 1]
```diff
@@ -46,6 +46,16 @@ func newFullRestoreCmd() *cobra.Command {
...
+				f, ferr := restore.NewFixFrom(r)
+				if ferr != nil {
+					return err
+				}
+
+				ferr = f.Fix()
+				if ferr != nil {
+					fmt.Println("Fix failed when restore failed", ferr)
+				}
+
```
Comment (critical27) [inline cmd/restore.go:58]: Do we need to print some log here? No matter fix succeeded or not, the restore failed anyway

Comment (pengweisong) [reply cmd/restore.go:58]: We will return the restore error below, and the outside code will print this err.

Comment (critical27) [reply cmd/restore.go:58]: 👌

**Review by critical27 — APPROVED** [review]
Good job~LGTM

**Pull request merged**